CODE_OF_CONDUCT.md

```diff
@@ -72,19 +72,19 @@ the consequences for any action they deem in violation of this Code of Conduct:
 
 ### 1. Correction
 
-**Community Impact**: Use of inappropriate language or other behavior deemed
```

Comment:
Actually this is correct spelling on behavior Ig?

Reply:
"behavior" is American English, "behaviour" is British English